demisto_sdk/commands/content_graph/interface/neo4j/queries/nodes.py

```diff
@@ -77,38 +77,47 @@ def create_nodes_by_type(
     logger.info(f'Created {nodes_count} nodes of type {content_type}.')
 
 
-def get_packs_content_items(
+def get_packs(
     tx: Transaction,
     marketplace: MarketplaceVersions,
-):
+    **properties,
+) -> List[Pack]:
+    params_str = to_neo4j_map(properties)
     query = f"""
-    MATCH (p:{ContentType.PACK})<-[:{Relationship.IN_PACK}]-(c:{ContentType.BASE_CONTENT})
-    WHERE '{marketplace}' IN p.marketplaces
+    MATCH (p:{ContentType.PACK}{params_str})<-[:{Relationship.IN_PACK}]-(c:{ContentType.BASE_CONTENT})
+    WHERE '{marketplace}' IN p.marketplaces AND '{marketplace}' IN c.marketplaces
     RETURN p AS pack, collect(c) AS content_items
     """
-    return run_query(tx, query).data()
-
-
-def get_all_integrations_with_commands(
+    packs: List[Pack] = []
+    integrations_to_commands = _get_all_integrations_with_commands(tx)
+    for item in run_query(tx, query).data():
+        pack = item.get('pack')
```

Comment:
First run the original query, and only when it succeeds run the `integrations_to_command` query.